lib/mix/tasks/chatkit_room.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Mix.Tasks.Cadet.ChatkitRoom do
+  @moduledoc """
+    Creates ChatKit rooms for answers with empty comments in the database.
+    Room creation: https://pusher.com/docs/chatkit/reference/api#create-a-room
+    Status codes: https://pusher.com/docs/chatkit/reference/api#response-and-error-codes
+
+    Note:
+    - Task is to run daily
+  """
+  use Mix.Task
+
+  import Ecto.Query
+  import Mix.EctoSQL
+
+  alias Cadet.Repo
+  alias Cadet.Assessments.Submission
+  alias Cadet.Chat.Room
+
+  def run(_args) do
+    ensure_started(Repo, [])
+    HTTPoison.start()
+
+    Submission
+    |> where(status: ^:submitted)
+    |> Repo.all()
+    |> Enum.each(fn submission -> Room.create_rooms(submission) end)
```

Comment:
```elixir
Submission
  |> join(:left, [s], a in assoc(s, :answers))
  |> preload([_, a], answers: a)
  |> where([_, a], is_nil(a.comment) or a.comment == "")
  |> Repo.all()
```

or since you have a function clause for answers, we can do

```elixir
Answer
  |> where([a], is_nil(a.comment) or a.comment == "")
  |> join(:left, [a], s in assoc(a, :submission))
  |> preload([_, s], submission: s)
  |> Repo.all()
```

Reply:
Done